util/stmtsummary/statement_summary.go

```diff
@@ -383,25 +405,68 @@ func (ssMap *stmtSummaryByDigestMap) SetRefreshInterval(value string, inSession
 			interval = 0
 		}
 	}
-	if interval > 0 {
-		atomic.StoreInt64(&ssMap.sysVars.refreshInterval, int64(interval))
+	// Users may set invalid values in configuration file.
+	if interval <= 0 {
```

Comment:
Can we move this config check into `config.Valid`?

Reply:
Great, done. But the global variable can also be '', so a default value is needed. So I use the config value as the default value.